Dockerfile

```diff
@@ -20,3 +20,5 @@ RUN set -e \
  && ln -s /src "$gem_dir" \
  && apt-get purge -y wget ruby-dev \
  && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+CMD exec ./submission.sh
```

Comment:
We can't change the behaviour of the `vmck/vagrant-vmck` image, so let's make another image for the ACS homeworks. Please create a repo `vmck/vagrant-vmck-acs`, that builds a docker image based on this image, that includes `submission.sh` and `Vagrantfile`, and only keep the `lib/vagrant-vmck/action/create.rb` changes here.